File: C++/2026-01-01/04_Input.cpp
#include <iostream>
// cout << output
// cin >> input
int main(){
    std::string name;
    std::string fullname;
    int age;
    std::cout << "What's your name?: ";
    std::cin >> name;
    std::cin.ignore();// or on std::getline(std::cin>>std::ws, )
    std::cout << "What's your full name?: ";
    std::getline(std::cin, fullname); //For string with " " space
    std::cout << "What's your age?: ";
    std::cin >> age;
    std::cout << "Hello "<<fullname<<'\n';
    std::cout << "Hello "<< name << '\n';
    std::cout << "You are " << age << " years old" << '\n';
    return 0;
};

File: Python/2025-12-30/04_Input.py
#input() = A function that prompts the use to enter data. Return the entered data as a string

name = input("What is your name? ")
print(f"Hello {name}")
age = input("How old are you?")
print(f"You are {age} yo")

name1 = input("What are your name again? ")
if bool(name1):
    print(f"You are {name1}")
else:
    name1 = input("What are your name again? ")